frontends/concrete-python/concrete/fhe/representation/node.py

```diff
@@ -473,7 +473,7 @@ def converted_to_table_lookup(self) -> bool:
         ]
 
     @property
-    def conversion_have_table_lookup(self) -> bool:
+    def conversion_have_table_lookup(self) -> bool:  # pragma: no cover # used only to draw graph
```

Comment:
Could you keep the pragma line here but move the comment to the documentation of the method?